Makefile

```diff
@@ -124,7 +114,7 @@ bench: fmt
 test:
 	go test -short -tags='debug testing netgo' -timeout=5s $(pkgs) -run=$(run) -count=$(count)
 
-test-long: lint lint-ci
+test-long: lint
 	@mkdir -p cover
 	GORACE='$(racevars)' go test -race --coverprofile='./cover/cover.out' -v -failfast -tags='testing debug netgo' -timeout=30s $(pkgs) -run=$(run) -count=$(count)
```

Comment:
This times out for me at 30s, so we should increase the timeout.

They are also failing when I do increase the timeout. @peterjan are they supposed to pass?